src/app/dialog/dialog.service.ts

```diff
@@ -48,9 +48,16 @@ export class DialogService {
     }
   }
 
+  public closeDialogWithoutRedirect() {
+    this.open = false;
```

Comment:
What about cleaning the callback? Would not it be better to add optional `skipRedirect` parameter to `closeDialog` method so we don't need to keep both these methods in sync when something is added to one of them?